web/src/RootSwitcher.tsx

```diff
@@ -8,6 +8,7 @@ import {
   createCitiesEndpoint,
   JPAL_TRACKING_ROUTE,
   LANDING_ROUTE,
+  LICENSE_ROUTE,
```

Comment:
Didn't recognize before but I think this should be `LICENSES_ROUTE` (plural), especially since its value is also `licenses`.

Reply:
renamed